cpu/cc2538/clock.c

```diff
@@ -61,8 +61,20 @@
 
 #include <stdint.h>
 /*---------------------------------------------------------------------------*/
-#define RTIMER_CLOCK_TICK_RATIO (RTIMER_SECOND / CLOCK_SECOND)
-#define RELOAD_VALUE (125000 - 1)  /** Fire 128 times / sec */
+#define RTIMER_CLOCK_TICK_RATIO			(RTIMER_SECOND / CLOCK_SECOND)
```

Comment:
The assigned value is too far on the right, exceeding the 80-char limit. Moreover, only 2 of the 3 definitions are aligned.